src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class ="body">
+    <div class="calendar calendar--start-day--mon calendar--month--30 body__calendar--size-modifier">
+      <div class="calendar__cells"></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
```
Not critical, but actually each cell is a singular